packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/addUser/addUser.routing.js

```diff
@@ -0,0 +1,54 @@
+import find from 'lodash/find';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.objects.objects.addUser',
```

Comment:
`objects.objects` ? Two lists of objects ?

Reply:
We have `objects` as entire view and empty url `''` that redirects to list of objects.
Can change the name of states in main PR.
Since this has multiple forks, can i do this refactoring once its merged?
It would be cleaner and easier to track this.

Reply:
Please take care of this in main PR. Its very confusing to have parent and child route with same name.

Reply:
Yes i agree. I will take care of this once i have 3 other child PRs merged to the `feat/add-s3-users `